devfile.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: 1.0.0
+
+metadata:
+  generateName: che-plugin-registry-dev-
+
+
+components:
+
+  - alias: didact-plugin
+    type: chePlugin
+    reference: https://che-plugin-registry.openshift.io/v3/plugins/redhat/vscode-didact/latest/meta.yaml
+    # id: redhat/vscode-didact/latest
+    preferences:
+      didact.openDefaultTutorialAtStartup: true
+      che.welcome.enable: false
+      didact.defaultUrl: /projects/che-plugin-registry/.che-didact-guide.md
+
+  - reference: https://che-plugin-registry.openshift.io/v3/plugins/ms-vscode/vscode-github-pullrequest/latest/meta.yaml
+    # id: ms-vscode/vscode-github-pullrequest/latest
+    type: chePlugin
+
+  - mountSources: true
+    memoryLimit: 2G
+    type: dockerimage
+    image: 'quay.io/eclipse/che-plugin-registry-dev:latest'
```

Comment:
What does this image contain?

Reply:
https://github.com/che-incubator/che-plugin-registry-dev-image it is a temporary image. We will use a classic node image as soon as we migrate the bash script to the node app that is performing the build

Reply:
Could we use already existent image with nodejs instead of this one? Let's say, it oculd be the image used in this devfile https://github.com/eclipse/che-devfile-registry/blob/master/devfiles/nodejs-yarn/devfile.yaml

Reply:
@vitaliy-guliy it requires all the other build tools like skopeo, jq, yq, python etc and is not yet only requiring nodejs only

Reply:
yes but at the moment we need these things: 
https://github.com/che-incubator/che-plugin-registry-dev-image/blob/main/Dockerfile#L14

```
RUN apk add --no-cache py3-pip jq bash git wget skopeo && pip3 install yq jsonschema
```

taken from https://github.com/eclipse/che-plugin-registry/blob/master/build/dockerfiles/Dockerfile#L15